Library/Formula/git-vendor.rb

```diff
@@ -0,0 +1,21 @@
+class GitVendor < Formula
+  desc "command for managing git vendored dependencies"
+  homepage "https://brettlangdon.github.io/git-vendor"
+  url "https://github.com/brettlangdon/git-vendor/archive/v1.1.0.tar.gz"
+  sha256 "1a4db75631af559152f19b41867c23c0fe6c4ff30117afa6ab2d02a5802b89e6"
+  head "https://github.com/brettlangdon/git-vendor.git"
+
+  def install
+    system "make", "PREFIX=#{prefix}", "install"
+  end
+
+  test do
+    system "git", "vendor", "--help"
+    system "git", "init"
+    system "git", "config", "user.email", "author@example.com"
+    system "git", "config", "user.name", "Au Thor"
+    system "git", "add", "."
+    system "git", "commit", "-m", "Initial commit"
+    system "git", "vendor", "add", "forge", "https://github.com/brettlangdon/forge", "v0.1.4"
+  end
```

Comment:
How about an `assert_match "forge@v0.1.4", shell_output("git vendor list")` to test reading the vendor list back out?